releasenotes/notes/Added_method_Instruction_is_parameterized-7a690119642f1dc1.yaml

```diff
@@ -0,0 +1,4 @@
+---
+features:
+  - |
+    Added method ``Instruction.is_parameterized()`` reducing repeated code.
```

Comment:
Thanks for adding a release note! Can you include that this returns true only if an instruction has a `ParameterExpression` or `Parameter` object for one of its `params`?

Reply:
Done.